### PR TITLE
SAAS-362 UI components tests

### PR DESCRIPTION
**Pull request by nicolalamacchia**

### CODE REVIEW

File: packages/platform-core/src/components/Checkbox/Checkbox.test.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React, { FC } from 'react';
+import { mount } from 'enzyme';
+import { CheckboxField } from './CheckboxField';
+import { Form, Field } from 'react-final-form';
+
+jest.spyOn(console, 'error').mockImplementation(() => {});
```
Comment (atymchuk) [inline packages/platform-core/src/components/Checkbox/Checkbox.test.tsx:6]: There is way to avoid putting this line in every test. 
1/ export console to /shared
2/ put a mock to its __mock__ folder
3/ use it without ^ that line :)

File: packages/platform-core/src/shared/validators/containsNumbers.test.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { containsNumber } from './containsNumber';
+
+describe('validators :: containsNumber', () => {
+  it('returns an error if there are no numbers characters in the passed string', () => {
+    expect(containsNumber('123')).toEqual(undefined);
```
Comment (atymchuk) [inline packages/platform-core/src/shared/validators/containsNumbers.test.ts:5]: We need to test edge cases by passing undefined, {}, [], '', etc
The validator may fail, btw. Then we should make it robust :)

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: packages/platform-core/src/shared/validators/email.test.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { email } from './email';
+
+describe('validators :: email', () => {
+  // NOTE (nicolalamacchia): some of these tests were taken from Chromium's source code
```
[the remark this answers is not in the document]
Comment (nicolalamacchia) [reply packages/platform-core/src/shared/validators/email.test.ts]: None taken, actually I agree with you, done!

File: packages/platform-core/src/components/Checkbox/Checkbox.test.tsx (hunk 1 above)
Comment (nicolalamacchia) [reply packages/platform-core/src/components/Checkbox/Checkbox.test.tsx:6]: we will introduce a better logging system, deprecating the use of `console`

File: packages/platform-core/src/shared/validators/containsNumbers.test.ts (hunk 2 above)
Comment (atymchuk) [reply packages/platform-core/src/shared/validators/containsNumbers.test.ts:5]: Ok, let's give it another iteration. Looks acceptable at this stage. :)

Comment (nicolalamacchia) [reply packages/platform-core/src/shared/validators/containsNumbers.test.ts:5]: We have to harden our validators first, and subsequently improve test coverage

File: packages/platform-ui/src/components/Authenticated/Authenticated.tsx [hunk 4]
```diff
@@ -24,7 +25,7 @@ export const Authenticated: FC = () => {
         <p>{Messages.authenticatedFirstLine}</p>
         <p>
           <em>
-            <b>{email}</b>
+            <b data-qa="user-email">{email}</b>
```
Comment (atymchuk) [inline packages/platform-ui/src/components/Authenticated/Authenticated.tsx:28]: 👍

**Review by atymchuk — APPROVED** [review]
